smart-contracts/contracts/PublicLock.sol

```diff
@@ -53,6 +53,14 @@ contract PublicLock is ILockCore, ERC165, IERC721, IERC721Receiver, Ownable {
     uint refundPenaltyDenominator
   );
 
+  event Destroy(
+    address lock,
+    uint balance,
+    address owner
```

Comment:
Should be indexed - so `address indexed owner`

Reply:
done.